includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -317,6 +320,41 @@ function restore_wgm_thumbnails( $document_type, $document ) {
 			add_filter( 'woocommerce_order_item_name', array( 'WGM_Product', 'add_thumbnail_to_order' ), 100, 3 );
 		}
 	}
+
+	/**
+	 * Adds invoice number filter to the search filters available in the admin order search.
+	 * 
+	 * @param array $options List of available filters.
+	 * 
+	 * @return array
+	 */
+	function hpos_admin_search_filters( array $options ): array {
+		$all_element_position = array_search( 'all', array_keys( $options ) );
+		$all_element = array_slice( $options, $all_element_position, $all_element_position );
```

Comment:
It might be slightly faster if you preserve 'all' in a variable and then unset it. What do you think?